README.md

```diff
@@ -6,19 +6,63 @@ Basic Flask backend that has CRUD (Create, Read, Update, Delete) endpoints for a
 
 Deployed version can be found at https://zothacks-2020-fundamentals.herokuapp.com/
 
-## Instructions
-
-1. Create and activate your Python Virtual Environment
-    ```
-    $ python3 -m venv env
-    $ source env/bin/activate
-    ```
-2. Install the project dependencies
-    ```
-    $ pip3 install -r requirements.txt
-    ```
-3. Run the project without a debugger attached
-    ```
-    $ python3 app.py
-    ```
-4. Check Postman Collection for Endpoint Documentation
+## Try it!
+
+**👩‍👧 Clone repository**
+```
+git clone https://github.com/hackuci/zothacks-2020-backend
+cd zothacks-2020-backend
+```
+
+**🐍 Create Python virtual environment**
+
+There are a good amount of depencies for this project -- it will be good practice to use a virtual environment, albeit not necessary.
+
+On macOS and Linux:
+
+```
+python3 -m virtualenv env
```

Comment:
I think this isnt right. It should be `python3 -m venv env`

Reply:
bump

Reply:
mb

Reply:
should be resolved